.github/workflows/run-tests.yml

```diff
@@ -8,15 +8,13 @@ jobs:
 
         strategy:
             matrix:
-                php: [7.4, 7.3]
-                laravel: [8.*, 7.*, 6.*]
+                php: [8.0, 8.1]
+                laravel: [8.*, 9.*]
```

Comment:
I've actually realised this weekend that in the `Install dependencies` step we require the laravel package (`composer require "laravel/framework:${{ matrix.laravel }}"`), but the `composer.json` actually specifies individual Illuminate packages. I think installing Laravel works because it would contain all those packages, but we could just limit that step to installing things we require.

Reply:
I think `TestBench` expects the whole Laravel framework to be present right? On our tests we depend on a fully booted application.

I was following what [Spatie does](https://github.com/spatie/laravel-backup/blob/main/.github/workflows/run-tests.yml) in some of their packages.